### PR TITLE
mv workflows to aspen

### PR DESCRIPTION
**Pull request by phoenixAja**
### Description

moved workflows to aspen directory (makes it easier to write tests and share fixtures)

#### Issue
no issue

### Test plan
GHA

### CODE REVIEW

**Comment by ttung** [comment]
Can you deploy to terraform?

File: src/backend/aspen/workflows/align_gisaid/align.sh [hunk 1]
```diff
@@ -59,7 +59,7 @@ fi
 end_time=$(date +%s)
 
 # create the objects
-entity_id=$(/aspen/.venv/bin/python /aspen/src/backend/workflows/align_gisaid/save.py                 \
+entity_id=$(/aspen/.venv/bin/python /aspen/src/backend/aspen/workflows/align_gisaid/save.py                 \
```
Comment (ttung) [inline src/backend/aspen/workflows/align_gisaid/align.sh:62]: can you fix the `\`

File: src/backend/aspen/workflows/ingest_gisaid/ingest.sh [hunk 2]
```diff
@@ -36,7 +36,7 @@ fi
 end_time=$(date +%s)
 
 # create the objects
-entity_id=$(/aspen/.venv/bin/python /aspen/src/backend/workflows/ingest_gisaid/save.py \
+entity_id=$(/aspen/.venv/bin/python /aspen/src/backend/aspen/workflows/ingest_gisaid/save.py \
                                     --aspen-workflow-rev "${aspen_workflow_rev}"  \
                                     --aspen-creation-rev "${aspen_creation_rev}"  \
                                     --start-time "${start_time}"                  \
```
Comment (ttung) [inline src/backend/aspen/workflows/ingest_gisaid/ingest.sh]: nitpicky: can you fix the `\`

File: src/backend/aspen/workflows/nextstrain_run/build_tree.sh [hunk 3]
```diff
@@ -22,13 +22,13 @@ git checkout FETCH_HEAD
...
-cp /aspen/src/backend/workflows/nextstrain_run/nextstrain_profile/* /ncov/my_profiles/aspen/
+cp /aspen/src/backend/aspen/workflows/nextstrain_run/nextstrain_profile/* /ncov/my_profiles/aspen/
 
 # dump the sequences, metadata, and builds.yaml for a run out to disk.
 aligned_gisaid_location=$(
-    /aspen/.venv/bin/python /aspen/src/backend/workflows/nextstrain_run/export.py  \
+    /aspen/.venv/bin/python /aspen/src/backend/aspen/workflows/nextstrain_run/export.py  \
                             --phylo-run-id "$1"                               \
                             --sequences /ncov/data/sequences_aspen.fasta      \
                             --metadata /ncov/data/metadata_aspen.tsv          \
```
Comment (ttung) [inline src/backend/aspen/workflows/nextstrain_run/build_tree.sh:34]: ditto.

File: src/backend/aspen/workflows/nextstrain_run/build_tree.sh [hunk 4]
```diff
@@ -65,7 +65,7 @@ fi
 end_time=$(date +%s)
 
 # create the objects
-/aspen/.venv/bin/python /aspen/src/backend/workflows/nextstrain_run/save.py               \
+/aspen/.venv/bin/python /aspen/src/backend/aspen/workflows/nextstrain_run/save.py               \
                         --aspen-workflow-rev "${aspen_workflow_rev}"                 \
                         --aspen-creation-rev "${aspen_creation_rev}"                 \
                         --ncov-rev "${ncov_git_rev}"                                 \
```
Comment (ttung) [inline src/backend/aspen/workflows/nextstrain_run/build_tree.sh]: ditto.

File: src/backend/aspen/workflows/transform_gisaid/transform.sh [hunk 5]
```diff
@@ -64,7 +64,7 @@ fi
 end_time=$(date +%s)
 
 # create the objects
-entity_id=$(/aspen/.venv/bin/python src/backend/workflows/transform_gisaid/save.py    \
+entity_id=$(/aspen/.venv/bin/python src/backend/aspen/workflows/transform_gisaid/save.py    \
                                     --aspen-workflow-rev "${aspen_workflow_rev}" \
                                     --aspen-creation-rev "${aspen_creation_rev}" \
                                     --ncov-ingest-rev "${ncov_ingest_git_rev}"   \
```
Comment (ttung) [inline src/backend/aspen/workflows/transform_gisaid/transform.sh]: ditto.

**Review by ttung — APPROVED** [review]
rest lgtm.

File: src/backend/aspen/workflows/ingest_gisaid/ingest.sh (hunk 2 above)
Comment (phoenixAja) [reply src/backend/aspen/workflows/ingest_gisaid/ingest.sh]: wait what's wrong with it?? 😂

File: src/backend/aspen/workflows/align_gisaid/align.sh (hunk 1 above)
Comment (phoenixAja) [reply src/backend/aspen/workflows/align_gisaid/align.sh:62]: do you want it to be in line with the previous `\`?

**Comment by phoenixAja** [comment]
> Can you deploy to terraform?

done!

File: src/backend/aspen/workflows/ingest_gisaid/ingest.sh (hunk 2 above)
Comment (ttung) [reply src/backend/aspen/workflows/ingest_gisaid/ingest.sh]: once upon a time, all the backslashes lined up.

**Comment by phoenixAja** [comment]
alright i formatted those slashes! I'll merge when the tests pass

**Pull request merged**